requirements/core-requirements.yaml

```diff
@@ -61,16 +61,6 @@ shap:
   minimum: "0.40"
   max_major_version: 0
 
-pandas-profiling:
-  pip_release: pandas-profiling
-  minimum: "3.1"
-  max_major_version: 3
```

Comment:
No longer needed, since we introduced a better profiling solution to MLP that doesn't rely on `pandas-profiling`